mappings/net/minecraft/client/network/packet/AdvancementUpdateClientPacket.mapping

```diff
@@ -0,0 +1,16 @@
+CLASS my net/minecraft/client/network/packet/AdvancementUpdateClientPacket
+	FIELD a clearCurrent Z
+	FIELD b toEarn Ljava/util/Map;
+	FIELD c toRemove Ljava/util/Set;
```

Comment:
`toLose`?

Reply:
If anything, toAdd. But I think at this size it's best to open a new issue for it..